src/Microsoft.Dism/DismException.cs

```diff
@@ -98,4 +101,17 @@ internal DismRebootRequiredException(int errorCode)
         {
         }
     }
-}
\ No newline at end of file
+
+    /// <summary>
+    /// The exception that is thrown when the previous operations requires a reload of image session.
+    /// </summary>
+    [Serializable]
+    public class DismReloadImageSessionRequiredException : DismException
```

Comment:
If there's a known set of method(s) in `DismApi` that will throw this, please add an `<exception cref="" />` like here: https://github.com/josemesona/ManagedDism/blob/dce273148640892935b52da75618ff291986435d/src/Microsoft.Dism/DismApi.cs#L252